repos/system_upgrade/el7toel8/actors/checkchrony/actor.py

```diff
@@ -0,0 +1,53 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import CalledProcessError, run
+
+
+class CheckChrony(Actor):
+    """
+    Check for incompatible changes in chrony configuration.
+
+    Warn that the default chrony configuration in RHEL8 uses the leapsectz
+    directive.
+    """
+
+    name = 'check_chrony'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report)
```

Comment:
missing trailing comma
```suggestion
    produces = (Report,)
```